hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -231,6 +231,12 @@
       en_run_modes: ["sw_test_mode"]
       reseed: 15
     }
+    {
+      name: chip_sw_rstmgr_sw_req
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["sw/device/tests/rstmgr_sw_req_test:1"]
```

Comment:
Awesome, thanks! Could you also add an entry in `chip_testplan.hjson` and link this test to it?

Reply:
o yeah good point i will do that